.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       xcode: "11.0.0"
     environment:
       - BAZEL_STARTUP_ARGS: "--output_base /Users/distiller/.cache/bazel"
-      - BAZEL_BUILD_ARGS: "--local_ram_resources=12288 --local_cpu_resources=8 --verbose_failures --test_env=ENVOY_IP_TEST_VERSIONS=v4only --test_output=all -c opt "
+      - BAZEL_BUILD_ARGS: "--local_ram_resources=12288 --local_cpu_resources=8 --verbose_failures --test_env=ENVOY_IP_TEST_VERSIONS=v4only --test_output=all "
```

Comment:
Why are you removing this? I believe that @bianpengyuan added it recently to work around the issues with the `DEBUG` define in some Google APIs when building on macOS.